pivotal-import/README.md

```diff
@@ -1,5 +1,7 @@
 This script imports a Pivotal Tracker CSV export file into a Shortcut workspace.
 
+[Walk-through Video](https://us06web.zoom.us/clips/share/b-GrMOkI75f5MPehYt_uK1RS7SdNy-ADfJqsew5gCQ5w05sNfAFhS56SgE4xIq2RIM-K68_9QNyJNVIwBb2sDkl3.ouSPQZoDx0WOD8ic) _(recorded 2024-04-05 using api-cookbook commit [1c12c1cc03](https://github.com/useshortcut/api-cookbook/tree/1c12c1cc035f4321f6b09a0e264eec740ddf2e88))_
```

Comment:
Not sure it matters, but I'd slightly prefer if this was in google drive.

Reply:
@opoku Good recommendation, I'll upload it there and replace the link. Thanks!

Reply:
@opoku Upon further inspection, I'll need to sort permissions issues on the company Google Drive side. I'm going to merge this so that it's out and available for users trying this out, but will add a Story to follow up with a Google Drive link when possible.